site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-list.vm

```diff
@@ -248,9 +248,7 @@
 									#end
 								</td>
 								<td headers="createdOn">
-									#set($createdTime="")
-									#set($createdTime = $!site.getCreatedTime())
-									$!createdTime.toStringLocalFull()
+									#if ($site.CreatedDate) $userTimeService.dateTimeFormat($site.CreatedDate, $tlang.Locale, 2) #end
```

Comment:
I'd just set this up in a variable, rather than passing the time service into the velocity render. We want to make these templates simpler, dumber, in the end.

Reply:
It's a site list.

Reply:
I see the issue. It just seems nasty passing services into the template render, but I see what you mean.

Reply:
I agree with Adrian about passing services into the template renderer it should be considered last resort as there are implications for doing this that are not apparent.

The best approach would be to create a PageBean that contains all the data need by the template Renderer.

However given that's a lot of work and out of scope this change seems reasonable.

Reply:
Yeah, I understand and agree with that view but in this case I feel like this was the way to go. Plus I also saw it on similar places.